### PR TITLE
Add dependencies (ncurses, libpcap) for nethogs

### PR DESCRIPTION
**Pull request by recolic**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Add missing dependencies for formula "nethogs". See file diff.

### CODE REVIEW

**Comment by Bo98** [comment]
Thanks @recolic!

Just some nitpicks that the CI picked up on:

```
==> brew audit nethogs --online
==> FAILED
Error: 2 problems in 1 formula detected
nethogs:
  * C: 8: col 3: dependency "libpcap" (line 8) should be put before dependency "ncurses" (line 7)
  * C: 10: col 3: `bottle` (line 10) should be put before `depends_on` (line 7)
```

File: Formula/nethogs.rb [hunk 1]
```diff
@@ -4,6 +4,9 @@ class Nethogs < Formula
   url "https://github.com/raboof/nethogs/archive/v0.8.5.tar.gz"
   sha256 "6a9392726feca43228b3f0265379154946ef0544c2ca2cac59ec35a24f469dcc"
 
+  depends_on "ncurses"
```
Comment (dawidd6) [inline Formula/nethogs.rb]:
```suggestion
  uses_from_macos "ncurses"
```

File: Formula/nethogs.rb [hunk 2]
```diff
@@ -4,6 +4,9 @@ class Nethogs < Formula
...
+  depends_on "libpcap"
```
Comment (dawidd6) [inline Formula/nethogs.rb]:
```suggestion
  uses_from_macos "libpcap"
```

**Comment by dawidd6** [comment]
Both dependencies are provided by macOS, so I suppose it would be wise to use `uses_from_macos` method instead of `depends_on`.

**Comment by recolic** [comment]
> Both dependencies are provided by macOS, so I suppose it would be wise to use `uses_from_macos` method instead of `depends_on`.

I fixed this problem.

btw, It seems that something wrong happened to the CI bot. I tried 'brew audit --strict nethogs' and it works.

**Comment by Bo98** [comment]
Well you fixed what it the bot was complaining about now so it should work now.

Note however GitHub is having issues so the CI's unable to get your new commit just now. We'll rerun the tests when things improve.

**Comment by Bo98** [comment]
@BrewTestBot test this please

**Comment by jonchang** [comment]
@BrewTestBot test this please

**Comment by jonchang** [comment]
Hi @recolic, thanks for your patience and thanks for contributing to Homebrew! I will merge this once CI passes.

**Pull request merged**